docs/providers/Serviceplatformen.md

```diff
@@ -0,0 +1,18 @@
+## Serviceplatformen provider
+Serviceplatformen primarily exhibits personal data. It is the responsibility of the municipality to ensure that this data is processed in accordance with 
+current legislation and thus the municipality's responsibility to ensure that via the Service Platform only access to the necessary personal data is given 
+for a specific IT system.
+
+For CPR purposes Logic connects to 'PersonBaseDataExtended' service [(LINK)](https://www.serviceplatformen.dk/administration/serviceOverview/show?uuid=e6be2436-bf35-4df2-83fe-925142825dc2). 
+Before Logic connection, client must firstly send the request for service agreement at STS Administration portal at specific environment (Test or Production). 
```

Comment:
Any chance we can provide a link to the STS Admin portal? Can we also talk about how and when to talk to the ServicePlatform help desk?

Reply:
Done